src/styl/_components/_breakingnews.styl

```diff
@@ -6,60 +6,61 @@
 // Markup: breakingnews.twig
 //
 // Styleguide: Components.BreakingNews
-/:root
-/[data-theme=light]
-/[data-theme] [data-theme=light]
-    --breakingnews-title-color: var(--color-highlight)
-
-/[data-theme=dark]
-/[data-theme] [data-theme=dark]
-    --breakingnews-title-color: var(--color-highlight-foreground)
 
 .breakingnews
     _fontload($fontstack-secondary)
     display flex
-    background var(--color-bg-primary)
+    background var(--color-highlight)
+    color var(--color-highlight-foreground)
     font-weight bold
     flex-wrap nowrap
     align-items stretch
     justify-content stretch
     border 0 // no border even it’s box
 
-    &-label
-    &-title
-    &-cta
+    &-content
         display flex
+        justify-content space-between
         align-items center
-        line-height 1.3em
+        width 100%
```

Comment:
What's the point to add this `&-content` and set its with to `100%`? Isn't the point to limit it to the `page-content` width?

Reply:
if i don't set a width right here , the content don't take all with of this element. In use `&-content` take the same size of `page-content.

Without the 100%
![Capture d’écran 2021-12-15 à 15 01 27](https://user-images.githubusercontent.com/39522234/146200305-6d4cd820-e48c-4a8f-a66a-ca7722f060ff.png)
`

Reply:
Sure.. Everything is in display flex..